coldfront/config/test_settings.py.sample

```diff
@@ -0,0 +1,142 @@
+REQUEST_APPROVAL_CC_LIST = ['test@test.test']
+
+
+#------------------------------------------------------------------------------
+# Billing settings
+#------------------------------------------------------------------------------
+
+
+#------------------------------------------------------------------------------
+# SSL settings
+#------------------------------------------------------------------------------
+
+# Use a secure cookie for the session cookie (HTTPS only).
+SESSION_COOKIE_SECURE = False
+
+#------------------------------------------------------------------------------
+# Sentry settings
+#------------------------------------------------------------------------------
+
+# Configure Sentry.
+import sentry_sdk
+from sentry_sdk.integrations.django import DjangoIntegration
+from sentry_sdk.integrations.logging import ignore_logger
+
+
+SENTRY_DSN = ''
+if SENTRY_DSN:
+    sentry_sdk.init(
+        dsn=SENTRY_DSN,
+        integrations=[DjangoIntegration()],
+        traces_sample_rate=0.01,
+        send_default_pii=True)
+    # Ignore noisy loggers.
+    ignore_logger('coldfront.api')
+    ignore_logger('coldfront.core.utils.middleware')
+
+#------------------------------------------------------------------------------
+# Django All-Auth settings
+#------------------------------------------------------------------------------
+
+CILOGON_APP_CLIENT_ID = ''
+CILOGON_APP_SECRET = ''
+
+#------------------------------------------------------------------------------
+# django-flags settings
+#------------------------------------------------------------------------------
+
+FLAGS = {
+    'ALLOCATION_RENEWAL_FOR_NEXT_PERIOD_REQUESTABLE': [
+        {'condition': 'during month', 'value': '5'},
+    ],
+    'BASIC_AUTH_ENABLED': [{'condition': 'boolean', 'value': True}],
+    'BRC_ONLY': [{'condition': 'boolean', 'value': True}],
+    'LRC_ONLY': [{'condition': 'boolean', 'value': False}],
+    'SECURE_DIRS_REQUESTABLE': [{'condition': 'boolean', 'value': True}],
+    'SERVICE_UNITS_PURCHASABLE': [{'condition': 'boolean', 'value': True}],
+    'SSO_ENABLED': [{'condition': 'boolean', 'value': False}],
+}
+
+#------------------------------------------------------------------------------
+# django-flags settings
+#------------------------------------------------------------------------------
+
+FLAGS = {
+    'ALLOCATION_RENEWAL_FOR_NEXT_PERIOD_REQUESTABLE': [
+        {'condition': 'during month', 'value': '5'},
+    ],
+    'BASIC_AUTH_ENABLED': [{'condition': 'boolean', 'value': True}],
+    'BRC_ONLY': [{'condition': 'boolean', 'value': True}],
+    'LRC_ONLY': [{'condition': 'boolean', 'value': False}],
+    'SECURE_DIRS_REQUESTABLE': [{'condition': 'boolean', 'value': True}],
+    'SERVICE_UNITS_PURCHASABLE': [{'condition': 'boolean', 'value': True}],
+    'SSO_ENABLED': [{'condition': 'boolean', 'value': False}],
+}
+
+#------------------------------------------------------------------------------
```

Comment:
I'm curious if this variable is needed (i.e., do any tests fail without them?).

Reply:
I've removed the variable

Reply:
I'm pretty sure we need to retain `FLAGS`.

Reply:
Yeah I retained FLAGS (there was a duplicate FLAGS that I had removed). I thought you were referring to Q_CLUSTER which didn't result in test fails so I removed it